udata_ods/migrations/2023-07-12-remove-double-slash-resource.py

```diff
@@ -0,0 +1,38 @@
+
+'''
+Remove ODS resources that are duplicate of resource with one slash only
+'''
+import logging
+
+from udata.models import Dataset
+from udata.harvest.models import HarvestSource
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing Harvest Sources.')
+
+    # sources with trailing slash
```

Comment:
Idk exactly how the URL is defined in there, is it trailing or heading slashes?

Reply:
All sources that have an url with a trailing slash (regex `/$`), with and ODS backend and created sooner than 2018-05-01.